data/hu.kramo.Cartridges.metainfo.xml.in

```diff
@@ -16,6 +16,23 @@
   <url type="contribute">https://github.com/kra-mo/cartridges/blob/main/CONTRIBUTING.md</url>
   <developer_name translatable="no">kramo</developer_name>
   <launchable type="desktop-id">@APP_ID@.desktop</launchable>
+  <translation type="gettext">cartridges</translation>
+  <categories>
+    <category>GNOME</category>
```

Comment:
I don't understand why these extra tags should be added. appstreamcli doesn't find any issues with the metainfo file currently and I would rather not duplicate strings.

Reply:
I forgot the related links. They are useful.

- The 'Translation' tag shows if apps are translated to relevant languages.
- Keywords provide extra search terms for software manager apps.
- Categories provide information about which categories the app should be listed in.

- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-categories
- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-keywords
- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-translation

Reply:
The translation tag I understand, but categories and keywords are already present in the .desktop file. Is there a reason to include them here too?

Reply:
Let me ask: https://gitlab.gnome.org/GNOME/gnome-software/-/issues/2375

Reply:
So I take that there is a reason to have the tags here too?

Reply:
Though it actually seems like Flathub populates these tags from the .desktop file anyway so maybe they aren't needed after all.

Reply:
"The .desktop files are available only if the app is installed, while distros create appstream data "caches" for available apps in the distro, thus users can install those apps (it's appstream-data package in Fedora). From that, at least for the Software, the main "source of truth" is the appstream data, not the .desktop files."

https://gitlab.gnome.org/GNOME/gnome-software/-/issues/2375#note_1909776

Reply:
I read that, yes. But that doesn't actually seem to be the case, at least for Flathub. I can still search for apps based on their tags even if they aren't installed. And given that tags are also present on [flathub.org](https://flathub.org), they probably extract the metadata from .desktop files automatically.

Reply:
My case was GNOME Software. Can you open a issue on Flathub and ask about keywords and categories tags. I'm going to ask to other desktops about this case.

Reply:
KDE - https://discuss.kde.org/t/does-kde-discover-parse-the-desktop-file-for-categories-and-keywords/7041

Reply:
KDE Discover also use these tags.

Reply:
I asked about it with regards to Flathub. They combine desktop and metainfo metadata with appstream-compose, you can see the result [here](https://hub.flathub.org/repo/appstream/x86_64/appstream.xml.gz). So there is no need for duplicating the keys, I will merge without those.